src/screens/ImportSeedPhraseSheet.js

```diff
@@ -283,16 +310,34 @@ export default function ImportSeedPhraseSheet() {
     hadPreviousAddressWithValue,
     handleSetImporting,
     hide,
+    goBack,
     initializeWallet,
     isImporting,
     name,
     navigate,
     resolvedAddress,
     seedPhrase,
+    selectedWallet.id,
+    selectedWallet.type,
     startAnalyticsTimeout,
+    wallets,
     wasImporting,
+    replace,
   ]);
 
+  useEffect(() => {
```

Comment:
This useEffect is duplicated in line 240 of this file (with the exception of using "walletLoadingStates.IMPORTING_WALLET" instead of "Importing...")